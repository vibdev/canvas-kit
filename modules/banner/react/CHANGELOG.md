# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.1.25](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.24...@workday/canvas-kit-react-banner@0.1.25) (2019-06-20)


### Bug Fixes

* broad stroke accessibility fixes ([9803412](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/commits/9803412))





## [0.1.24](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.23...@workday/canvas-kit-react-banner@0.1.24) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.23](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.22...@workday/canvas-kit-react-banner@0.1.23) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.22](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.21...@workday/canvas-kit-react-banner@0.1.22) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.21](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.20...@workday/canvas-kit-react-banner@0.1.21) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.20](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.19...@workday/canvas-kit-react-banner@0.1.20) (2019-06-14)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.19](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.18...@workday/canvas-kit-react-banner@0.1.19) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.18](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.17...@workday/canvas-kit-react-banner@0.1.18) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.17](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.16...@workday/canvas-kit-react-banner@0.1.17) (2019-06-10)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.16](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.15...@workday/canvas-kit-react-banner@0.1.16) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.15](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.14...@workday/canvas-kit-react-banner@0.1.15) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.14](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.13...@workday/canvas-kit-react-banner@0.1.14) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.13](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.12...@workday/canvas-kit-react-banner@0.1.13) (2019-05-31)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.12](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.11...@workday/canvas-kit-react-banner@0.1.12) (2019-05-24)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.11](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.10...@workday/canvas-kit-react-banner@0.1.11) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.9...@workday/canvas-kit-react-banner@0.1.10) (2019-05-21)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.8...@workday/canvas-kit-react-banner@0.1.9) (2019-05-15)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.7...@workday/canvas-kit-react-banner@0.1.8) (2019-05-14)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.6...@workday/canvas-kit-react-banner@0.1.7) (2019-05-10)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.5...@workday/canvas-kit-react-banner@0.1.6) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.4...@workday/canvas-kit-react-banner@0.1.5) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.3...@workday/canvas-kit-react-banner@0.1.4) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.2...@workday/canvas-kit-react-banner@0.1.3) (2019-05-07)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.1...@workday/canvas-kit-react-banner@0.1.2) (2019-05-06)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## 0.1.1 (2019-04-30)

**Note:** Version bump only for package @workday/canvas-kit-react-banner
