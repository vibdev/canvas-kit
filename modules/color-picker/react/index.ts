import ColorInput from './lib/ColorInput';
import ColorPreview from './lib/ColorPreview';

export default ColorInput;
export {ColorInput, ColorPreview};
export * from './lib/ColorInput';
export * from './lib/ColorPreview';
