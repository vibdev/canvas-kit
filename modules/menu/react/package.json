{
  "name": "@workday/canvas-kit-react-menu",
  "version": "0.0.24",
  "description": "A container for navigation or action items",
  "author": "Workday, Inc. (https://www.workday.com)",
  "license": "Apache-2.0",
  "main": "dist/commonjs/index.js",
  "module": "dist/es6/index.js",
  "sideEffects": false,
  "types": "dist/es6/index.d.ts",
  "repository": {
    "type": "git",
    "url": "https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu"
  },
  "files": [
    "dist/",
    "lib/",
    "index.ts"
  ],
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "clean": "rimraf dist && rimraf .build-info && mkdirp dist",
    "build:cjs": "tsc -p tsconfig.cjs.json",
    "build:es6": "tsc -p tsconfig.es6.json",
    "build:rebuild": "npm-run-all clean build",
    "build": "npm-run-all --parallel build:cjs build:es6"
  },
  "keywords": [
    "canvas",
    "canvas-kit",
    "react",
    "components",
    "workday",
    "menu"
  ],
  "devDependencies": {
    "@material-ui/core": "^3.9.2",
    "@types/react-transition-group": "^2.0.14",
    "@workday/canvas-system-icons-web": "^1.0.1",
    "react-transition-group": "^2.5.0"
  },
  "dependencies": {
    "@types/uuid": "^3.4.4",
    "@workday/canvas-kit-react-card": "^0.3.23",
    "@workday/canvas-kit-react-common": "^0.7.5",
    "@workday/canvas-kit-react-core": "^1.10.6",
    "@workday/canvas-kit-react-icon": "^1.7.7",
    "@workday/design-assets-types": "^0.2.3",
    "emotion": "^9.2.12",
    "react-emotion": "^9.2.12",
    "uuid": "^3.3.2"
  },
  "peerDependencies": {
    "react": ">= 16.3 < 17"
  }
}
