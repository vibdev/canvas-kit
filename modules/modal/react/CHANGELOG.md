# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.0.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/modal/compare/@workday/canvas-kit-react-modal@0.0.1...@workday/canvas-kit-react-modal@0.0.2) (2019-06-20)

**Note:** Version bump only for package @workday/canvas-kit-react-modal





## 0.0.1 (2019-06-20)

**Note:** Version bump only for package @workday/canvas-kit-react-modal
