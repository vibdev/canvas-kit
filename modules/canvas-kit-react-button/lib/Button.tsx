import * as React from 'react';
import {ButtonBaseCon, ButtonBaseLabel, ButtonDataLabel, ButtonIconLabel} from './ButtonBase';
import {ButtonTypes, ButtonSizes, ButtonIconPositions} from './types';
import {GrowthBehavior} from '@workday/canvas-kit-react-common';

export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement>, GrowthBehavior {
  /**
   * Button cannot be empty.
   */
  children: React.ReactNode;
  /**
   * Type of button.
   */
  buttonType?: ButtonTypes;
  /**
   * Size of button.
   */
  buttonSize?: ButtonSizes;
  /**
   * Ref of button that the styled component renders.
   */
  buttonRef?: React.Ref<HTMLButtonElement>;
  /**
   * Data label of button.
   */
  dataLabel?: String;
  /**
   * Left positioned icon for button.
   */
  leftIcon?: React.ReactNode;
  /**
   * Right positioned icon button.
   */
  rightIcon?: React.ReactNode;
}

export default class Button extends React.Component<ButtonProps> {
  public static Types = ButtonTypes;
  public static Sizes = ButtonSizes;

  static defaultProps: Partial<ButtonProps> = {
    buttonSize: ButtonSizes.Large,
    buttonType: ButtonTypes.Secondary,
    grow: false,
    dataLabel: false,
    leftIcon: false,
    rightIcon: false,
  };

  public render() {
    const {buttonRef, ...elemProps} = this.props;

    return (
      <ButtonBaseCon {...elemProps} innerRef={buttonRef}>
        {elemProps.leftIcon && (
          <ButtonIconLabel
            iconPosition={ButtonIconPositions.Left}
            icon={elemProps.leftIcon}
            {...elemProps}
          />
        )}
        <ButtonBaseLabel buttonSize={elemProps.buttonSize}>{elemProps.children}</ButtonBaseLabel>
        {elemProps.dataLabel && (
          <ButtonDataLabel {...elemProps}>{elemProps.dataLabel}</ButtonDataLabel>
        )}
        {elemProps.rightIcon && (
          <ButtonIconLabel
            iconPosition={ButtonIconPositions.Right}
            icon={elemProps.rightIcon}
            {...elemProps}
          />
        )}
      </ButtonBaseCon>
    );
  }
}
