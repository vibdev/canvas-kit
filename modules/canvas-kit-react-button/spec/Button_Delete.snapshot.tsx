import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Button from '../lib/Button';

describe('Button (Delete) Snapshots', () => {
  test('renders a large, delete button', () => {
    const component = renderer.create(
      <Button buttonSize={Button.Sizes.Large} buttonType={Button.Types.Delete}>
        Button
      </Button>
    );
    expect(component).toMatchSnapshot();
  });

  test('renders a medium, delete button', () => {
    const component = renderer.create(
      <Button buttonSize={Button.Sizes.Medium} buttonType={Button.Types.Delete}>
        Button
      </Button>
    );
    expect(component).toMatchSnapshot();
  });

  test('renders a small, delete button', () => {
    const component = renderer.create(
      <Button buttonSize={Button.Sizes.Small} buttonType={Button.Types.Delete}>
        Button
      </Button>
    );
    expect(component).toMatchSnapshot();
  });

  test('renders a large, UDE delete button', () => {
    const component = renderer.create(
      <Button buttonSize={Button.Sizes.Large} buttonType={Button.Types.UdeDelete}>
        Button
      </Button>
    );
    expect(component).toMatchSnapshot();
  });

  test('renders a medium, UDE delete button', () => {
    const component = renderer.create(
      <Button buttonSize={Button.Sizes.Medium} buttonType={Button.Types.UdeDelete}>
        Button
      </Button>
    );
    expect(component).toMatchSnapshot();
  });

  test('renders a small, UDE delete button', () => {
    const component = renderer.create(
      <Button buttonSize={Button.Sizes.Small} buttonType={Button.Types.UdeDelete}>
        Button
      </Button>
    );
    expect(component).toMatchSnapshot();
  });
});
