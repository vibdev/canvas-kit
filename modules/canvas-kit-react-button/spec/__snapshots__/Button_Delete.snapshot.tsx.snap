// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button (Delete) Snapshots renders a large, UDE delete button 1`] = `
.emotion-1 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 13px;
  border-radius: 999px;
  border: 2px solid transparent;
  box-shadow: none;
  position: relative;
  cursor: pointer;
  outline: none;
  -webkit-transition: all 120ms ease-in;
  transition: all 120ms ease-in;
  background-color: #de2e21;
  border-color: #de2e21;
  color: #ffffff;
  min-width: 112px;
  height: 48px;
  padding: 0 18px;
}

.emotion-1:hover:active {
  -webkit-transition-duration: 40ms;
  transition-duration: 40ms;
}

.emotion-1:disabled,
.emotion-1:disabled:active {
  cursor: default;
  box-shadow: none;
}

.emotion-1:focus {
  background-color: #de2e21;
  color: #ffffff;
}

.emotion-1:hover:focus {
  background-color: #a31b12;
}

.emotion-1:active,
.emotion-1:focus:active,
.emotion-1:hover:active {
  background-color: #80160E;
  border-color: transparent;
  color: #ffffff;
}

.emotion-1:hover {
  background-color: #a31b12;
  border-color: #a31b12;
  color: #ffffff;
}

.emotion-1:disabled,
.emotion-1:active:disabled,
.emotion-1:focus:disabled,
.emotion-1:hover:disabled {
  background-color: #fcc9c5;
  border-color: transparent;
  color: #ffffff;
}

.emotion-1:not([disabled]):focus,
.emotion-1:not([disabled]):active {
  -webkit-animation: animation-72u8ku 100ms;
  animation: animation-72u8ku 100ms;
  box-shadow: 0 0 0 2px #ffffff,0 0 0 4px #005cb9;
}

.emotion-0 {
  position: relative;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  font-weight: 700;
  font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 16px;
  padding: 0 12px;
}

.emotion-0:hover:active {
  background-color: transparent;
}

<button
  className="emotion-1"
>
  <span
    className="emotion-0"
  >
    Button
  </span>
</button>
`;

exports[`Button (Delete) Snapshots renders a large, delete button 1`] = `
.emotion-1 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 14px;
  border-radius: 999px;
  border: 1px solid transparent;
  box-shadow: none;
  position: relative;
  cursor: pointer;
  outline: none;
  -webkit-transition: all 120ms ease-in;
  transition: all 120ms ease-in;
  background-color: #ffefee;
  border-color: #ff5347;
  color: #de2e21;
  height: 40px;
  padding: 0 32px;
  min-width: 112px;
  max-width: 288px;
  font-weight: 500;
}

.emotion-1:hover:active {
  -webkit-transition-duration: 40ms;
  transition-duration: 40ms;
}

.emotion-1:disabled,
.emotion-1:disabled:active {
  cursor: default;
  box-shadow: none;
}

.emotion-1:focus {
  background-color: #ffefee;
  border-color: #005cb9;
  color: #de2e21;
}

.emotion-1:hover:focus {
  background-color: #ff5347;
}

.emotion-1:active,
.emotion-1:focus:active,
.emotion-1:hover:active {
  background-color: #ffefee;
  border-color: #ff5347;
  color: #de2e21;
}

.emotion-1:hover {
  background-color: #ff5347;
  border-color: #a31b12;
  color: #ffffff;
}

.emotion-1:disabled,
.emotion-1:active:disabled,
.emotion-1:focus:disabled,
.emotion-1:hover:disabled {
  background-color: #ffefee;
  border-color: #fcc9c5;
  color: #ff867d;
}

.emotion-1:not([disabled]):focus,
.emotion-1:not([disabled]):active {
  border-color: #005cb9;
  -webkit-animation: animation-72u8ku 100ms;
  animation: animation-72u8ku 100ms;
  box-shadow: 0 0 0 2px #ffffff,0 0 0 4px #005cb9;
}

.emotion-0 {
  position: relative;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  font-weight: 700;
  font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 16px;
  padding: 0 12px;
  font-size: inherit;
  font-weight: inherit;
  padding: 0px;
}

.emotion-0:hover:active {
  background-color: transparent;
}

<button
  className="emotion-1"
>
  <span
    className="emotion-0"
  >
    Button
  </span>
</button>
`;

exports[`Button (Delete) Snapshots renders a medium, UDE delete button 1`] = `
.emotion-1 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 13px;
  border-radius: 999px;
  border: 2px solid transparent;
  box-shadow: none;
  position: relative;
  cursor: pointer;
  outline: none;
  -webkit-transition: all 120ms ease-in;
  transition: all 120ms ease-in;
  background-color: #de2e21;
  border-color: #de2e21;
  color: #ffffff;
  min-width: 96px;
  height: 40px;
  padding: 0 14px;
}

.emotion-1:hover:active {
  -webkit-transition-duration: 40ms;
  transition-duration: 40ms;
}

.emotion-1:disabled,
.emotion-1:disabled:active {
  cursor: default;
  box-shadow: none;
}

.emotion-1:focus {
  background-color: #de2e21;
  color: #ffffff;
}

.emotion-1:hover:focus {
  background-color: #a31b12;
}

.emotion-1:active,
.emotion-1:focus:active,
.emotion-1:hover:active {
  background-color: #80160E;
  border-color: transparent;
  color: #ffffff;
}

.emotion-1:hover {
  background-color: #a31b12;
  border-color: #a31b12;
  color: #ffffff;
}

.emotion-1:disabled,
.emotion-1:active:disabled,
.emotion-1:focus:disabled,
.emotion-1:hover:disabled {
  background-color: #fcc9c5;
  border-color: transparent;
  color: #ffffff;
}

.emotion-1:not([disabled]):focus,
.emotion-1:not([disabled]):active {
  -webkit-animation: animation-72u8ku 100ms;
  animation: animation-72u8ku 100ms;
  box-shadow: 0 0 0 2px #ffffff,0 0 0 4px #005cb9;
}

.emotion-0 {
  position: relative;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  font-weight: 700;
  font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 14px;
  padding: 0 8px;
}

.emotion-0:hover:active {
  background-color: transparent;
}

<button
  className="emotion-1"
>
  <span
    className="emotion-0"
  >
    Button
  </span>
</button>
`;

exports[`Button (Delete) Snapshots renders a medium, delete button 1`] = `
.emotion-1 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 13px;
  border-radius: 999px;
  border: 1px solid transparent;
  box-shadow: none;
  position: relative;
  cursor: pointer;
  outline: none;
  -webkit-transition: all 120ms ease-in;
  transition: all 120ms ease-in;
  background-color: #ffefee;
  border-color: #ff5347;
  color: #de2e21;
  height: 24px;
  padding: 0 24px;
  min-width: 80px;
  max-width: 200px;
  font-weight: 500;
}

.emotion-1:hover:active {
  -webkit-transition-duration: 40ms;
  transition-duration: 40ms;
}

.emotion-1:disabled,
.emotion-1:disabled:active {
  cursor: default;
  box-shadow: none;
}

.emotion-1:focus {
  background-color: #ffefee;
  border-color: #005cb9;
  color: #de2e21;
}

.emotion-1:hover:focus {
  background-color: #ff5347;
}

.emotion-1:active,
.emotion-1:focus:active,
.emotion-1:hover:active {
  background-color: #ffefee;
  border-color: #ff5347;
  color: #de2e21;
}

.emotion-1:hover {
  background-color: #ff5347;
  border-color: #a31b12;
  color: #ffffff;
}

.emotion-1:disabled,
.emotion-1:active:disabled,
.emotion-1:focus:disabled,
.emotion-1:hover:disabled {
  background-color: #ffefee;
  border-color: #fcc9c5;
  color: #ff867d;
}

.emotion-1:not([disabled]):focus,
.emotion-1:not([disabled]):active {
  border-color: #005cb9;
  -webkit-animation: animation-72u8ku 100ms;
  animation: animation-72u8ku 100ms;
  box-shadow: 0 0 0 2px #ffffff,0 0 0 4px #005cb9;
}

.emotion-0 {
  position: relative;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  font-weight: 700;
  font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 14px;
  padding: 0 8px;
  font-size: inherit;
  font-weight: inherit;
  padding: 0px;
}

.emotion-0:hover:active {
  background-color: transparent;
}

<button
  className="emotion-1"
>
  <span
    className="emotion-0"
  >
    Button
  </span>
</button>
`;

exports[`Button (Delete) Snapshots renders a small, UDE delete button 1`] = `
.emotion-1 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 13px;
  border-radius: 999px;
  border: 2px solid transparent;
  box-shadow: none;
  position: relative;
  cursor: pointer;
  outline: none;
  -webkit-transition: all 120ms ease-in;
  transition: all 120ms ease-in;
  background-color: #de2e21;
  border-color: #de2e21;
  color: #ffffff;
  min-width: 80px;
  height: 32px;
  padding: 0 14px;
}

.emotion-1:hover:active {
  -webkit-transition-duration: 40ms;
  transition-duration: 40ms;
}

.emotion-1:disabled,
.emotion-1:disabled:active {
  cursor: default;
  box-shadow: none;
}

.emotion-1:focus {
  background-color: #de2e21;
  color: #ffffff;
}

.emotion-1:hover:focus {
  background-color: #a31b12;
}

.emotion-1:active,
.emotion-1:focus:active,
.emotion-1:hover:active {
  background-color: #80160E;
  border-color: transparent;
  color: #ffffff;
}

.emotion-1:hover {
  background-color: #a31b12;
  border-color: #a31b12;
  color: #ffffff;
}

.emotion-1:disabled,
.emotion-1:active:disabled,
.emotion-1:focus:disabled,
.emotion-1:hover:disabled {
  background-color: #fcc9c5;
  border-color: transparent;
  color: #ffffff;
}

.emotion-1:not([disabled]):focus,
.emotion-1:not([disabled]):active {
  -webkit-animation: animation-72u8ku 100ms;
  animation: animation-72u8ku 100ms;
  box-shadow: 0 0 0 2px #ffffff,0 0 0 4px #005cb9;
}

.emotion-0 {
  position: relative;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  font-weight: 700;
  font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 14px;
  padding: 0px;
}

.emotion-0:hover:active {
  background-color: transparent;
}

<button
  className="emotion-1"
>
  <span
    className="emotion-0"
  >
    Button
  </span>
</button>
`;

exports[`Button (Delete) Snapshots renders a small, delete button 1`] = `
.emotion-1 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 10px;
  border-radius: 999px;
  border: 1px solid transparent;
  box-shadow: none;
  position: relative;
  cursor: pointer;
  outline: none;
  -webkit-transition: all 120ms ease-in;
  transition: all 120ms ease-in;
  background-color: #ffefee;
  border-color: #ff5347;
  color: #de2e21;
  height: 18px;
  padding: 0 8px;
  min-width: 56px;
  max-width: 120px;
  font-weight: 500;
}

.emotion-1:hover:active {
  -webkit-transition-duration: 40ms;
  transition-duration: 40ms;
}

.emotion-1:disabled,
.emotion-1:disabled:active {
  cursor: default;
  box-shadow: none;
}

.emotion-1:focus {
  background-color: #ffefee;
  border-color: #005cb9;
  color: #de2e21;
}

.emotion-1:hover:focus {
  background-color: #ff5347;
}

.emotion-1:active,
.emotion-1:focus:active,
.emotion-1:hover:active {
  background-color: #ffefee;
  border-color: #ff5347;
  color: #de2e21;
}

.emotion-1:hover {
  background-color: #ff5347;
  border-color: #a31b12;
  color: #ffffff;
}

.emotion-1:disabled,
.emotion-1:active:disabled,
.emotion-1:focus:disabled,
.emotion-1:hover:disabled {
  background-color: #ffefee;
  border-color: #fcc9c5;
  color: #ff867d;
}

.emotion-1:not([disabled]):focus,
.emotion-1:not([disabled]):active {
  border-color: #005cb9;
  -webkit-animation: animation-72u8ku 100ms;
  animation: animation-72u8ku 100ms;
  box-shadow: 0 0 0 2px #ffffff,0 0 0 4px #005cb9;
}

.emotion-0 {
  position: relative;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  font-weight: 700;
  font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 14px;
  padding: 0px;
  font-size: inherit;
  font-weight: inherit;
  padding: 0px;
}

.emotion-0:hover:active {
  background-color: transparent;
}

<button
  className="emotion-1"
>
  <span
    className="emotion-0"
  >
    Button
  </span>
</button>
`;
